playbooks/byo/openshift-glusterfs/config.yml

```diff
@@ -0,0 +1,6 @@
+---
+- include: ../../common/openshift-cluster/std_include.yml
```

Comment:
For any entry point playbook, the need to include the following to build the appropriate groups.  This also picks up the playbook for syntax checking in CI.
```
- include: ../openshift-cluster/initialize_groups.yml
```